### DISCUSSION TITLE
Opinião de vocês na minha carreira

### DISCUSSION THREAD

**Discussion by guilopes22** [frontendbr/forum#961] [Discussão]
Fala galera, beleza?

Atualmente trabalho com PHP puro, jQuery e afins, porém isso não me agrada nem um pouco.

Gostaria de trabalhar com uma stack mais atualizada, venho estudando React, React Native, Relay, GraphQL e afins e decidi que o que eu gosto é isso, mobile, JS e tal, porém onde eu moro, é difícil encontrar empresas que utilizam essas tecnologias.

Venho diariamente estudando em casa após o horário de serviço e procurando/mandando currículos para vagas em empresas que aceitam remoto e/ou sejam aqui perto.

Comecei a mandar currículo para algumas empresas de outras cidades, porém, a julgar que sou júnior ou até menos devido ao meu grau de experiência com essas tecnologias, eu entraria ganhando pouco e não conseguiria pagar carro, aluguel, etc.

Está para abrir uma vaga na empresa que eu trabalho de mobile, porém para trabalhar com Ionic (nem possibilidade de mudar, o pessoal não gosta de arriscar por aqui). Só que aqui é meio desorganizado e tal e Ionic não me chama muito atenção, por mais que utilize Angular 5 e tal.

Qualquer sugestão, dica, se já passaram por situação parecida, como acham que eu devo proceder, podem falar aqui... Estou num grande dilema.

Agradeço desde já.

**Comment by yurischulz** [comment]
Fala cara, tudo certo?

Sinceramente acho que o seu nível de conhecimento em determinada tecnologia não é o fator principal na hora da contratação, o que mais importa é se você é um bom programador. Tecnologia se aprende (claro que isso é relativo à necessidade da empresa).

Trabalhar com Ionic / Angular 5 na sua empresa atual é uma boa oportunidade de adquirir experiência e largar o jQuery, em que pese não seja a sua tecnologia favorita, é uma bagagem que poderá te ajudar lá na frente, além de pagar as contas! hehehe

Assim como mudar de ares e encarar trabalhar com o que você tem estudado, também é válido! A falta de experiência e a necessidade de executar as tarefas funciona como um impulsionador e te obriga a se esforçar o dobro. Sou dessa opinião, a necessidade nos faz evoluir!

Meu conselho seria: Seja paciente e segue buscando vagas em que você possa trabalhar com o que quer e que proporcione o sustento que você precisa. Enquanto isso, segura as pontas por aí e segue estudando :)

Desenvolve projetos pessoais e alimenta seu Github com eles!
Paz!

**Edit:**
Avalia o seu momento pessoal e profissional, veja se é o momento de arriscar ou se é melhor ter o pé no chão e planejar melhor essa troca de ambiente.

**Comment by rubenmarcus** [comment]
Faça projetos pessoais em produção na tecnologia que você procurar trabalhar, e que exija profundo dominio técnico sobre o que você está fazendo , além de projetinhos de curso, faça um projeto real para necessidade real, quem sabe ele não vira seu emprego, ou serve de case para você conseguir seu trabalho remoto..

Pelo menos eu estou com essa mentalidade esse ano.. Entrar numa vaga é bom, mas esperar também alguem te dar uma oportunidade é uma merda..

acho que está num bom caminho, pague as contas primeiro, se precisar aprenda Ionic (não tenha preconceito com tecnologias) , eu mesmo queria aprender e trabalhar com Vue, mas surgiu oportunidade de trabalhar com Angular, e nunca fui de negar trabalho ou não aprender, por fora vou fazendo o que gosto com Vue e tentando virar algo real também

**Comment by rubenmarcus** [comment]
Outra forma é, ir atrás de freelas nas tecnologias que você quer, ou conseguir freelas mais simples, tipo um app pra algum negócio na sua cidade, e fazer na tecnologia e ir agregando ao portfolio... ajudar a construir libs... opensource..

**Comment by FerreiraRaphael** [comment]
Evolua sua conta no github com esses seus estudos, você disse que estuda essas novas tecnologias todo dia após o expediente, mas só criou uma conta no github hoje, você poderia estar mostrando o código que esta produzindo e estudando diariamente aqui, no github.

Isso ira comprovar sua experiencia e o mais importante e mostrar seu código para quem estiver te procurando. Não se preocupe com código feio no seu github, se preocupe em evoluir ele, pois quem ver seu código vai perceber a sua evolução.

Continue estudando e fazendo commits que rapidamente encontrara novas oportunidades.

**Comment by felipefialho** [comment]
Em cima do que o @FerreiraRaphael disse, [da uma lida nesse artigo](https://www.felipefialho.com/blog/2015/aproximando-seus-estudos-de-cenarios-reais), penso que pode te ajudar.
